### ISSUE TITLE
tutorial to use in visual studio?

### ISSUE THREAD

**Issue by franciscogimeno2000** [wokwi/wokwi-features#564]
Hello, does anyone know where I can find a tutorial from 0 on how to use Wokwi in visual studio Code? I am unable to get it to work and can't find anything. Thank you

**Comment by urish** [comment]
@matititam can you please assist here?

**Comment by franciscogimeno2000** [comment]
Hello, do you know where there is information from 0 to use the extension?
All examples and videos are for use on the web

**Comment by urish** [comment]
The documentation is at https://docs.wokwi.com/vscode/getting-started, and there's some video there - but it's not very comprehensive. Still, may be somewhat helpful.

**Comment by franciscogimeno2000** [comment]
I've seen it and it doesn't explain anything to me. I'm sorry I'm clumsy. You only see the code loaded and not how it got to that point.

**Comment by franciscogimeno2000** [comment]
Could some Wokwi technician make an explanatory video, please... It costs you 10 minutes!

**Comment by urish** [comment]
Yes the video there is not that great. Let's wait to see what @matititam can come up with.

**Comment by franciscogimeno2000** [comment]
I use Visual Studio and I've never used Visual Studio Code... I don't know, it's a very strange application to handle.

**Comment by matititam** [comment]
@franciscogimeno2000 I had played with some projects on VSCode.. I will try to create a video showing how I get the Wokwi Running on VSCode.. Please expect it by tomorrow :)

**Comment by franciscogimeno2000** [comment]
OK, thank you very much for your help. wait.

**Comment by franciscogimeno2000** [comment]
> @franciscogimeno2000Había jugado con algunos proyectos en VSCode. Intentaré crear un video que muestre cómo hago que Wokwi se ejecute en VSCode. Espéralo para mañana :)


Hello, how is the tutorial going?

**Comment by matititam** [comment]
It's cooking. Just a couple of hours. It will be ready

**Comment by franciscogimeno2000** [comment]
ok

**Comment by matititam** [comment]
Please have a look at this document. 
https://docs.google.com/document/d/1XQ6oEajXWh64AKem3sj6RQL7GPWH-MVa7xeWXK2pB6A/edit?usp=sharing

Share your feedback.

**Comment by franciscogimeno2000** [comment]
OK thanks for the help.

Some questions.

How do you make it stop at a stopping point?

How are the values ​​of the variables monitored?

Could I load a *.ino file?

**Comment by matititam** [comment]
To load an ino file, you will need Arduino Extension installed. 

https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.vscode-arduino

Project: https://github.com/wokwi/arduino-lcd-helloworld

(download, Unzip, Load folder.. ) and follow the instructions in the ReadMe file. 

I had some hiccups setting it up. I will get back to you. Meanwhile, you, too, can give it a try. 


Regarding stopping at a particualr point -> It is not possible at this point. Same goes with monitoring variables (unless you prinr themout). I might be wrong ehre and I am positive that there are some solutions from @urish

**Comment by franciscogimeno2000** [comment]
Opps... wow, I thought you could see and change the value of the variables, as well as put breakpoints in the program, without it being necessary to physically connect a micro.
